android-project/app/src/main/java/oceantreasur/es/fragments/ActivityMain.java

```diff
@@ -1,144 +1,152 @@
-        moveAnimation();
+        MainFragment mainFragment = new MainFragment();
+        attachFragment(mainFragment, "MAIN_FRAGMENT");
 
     }
 
-    public void attachFragment() {
-
+    public void attachFragment(android.app.Fragment fragment, String tag) {
+        fragmentTransaction = getFragmentManager().beginTransaction();
+        fragmentTransaction.replace(R.id.container, fragment, tag)
+                            .commit();
     }
 
-    private void inflateFishViews() {
-        RelativeLayout background = (RelativeLayout) findViewById(R.id.background);
-        RelativeLayout foreground = (RelativeLayout) findViewById(R.id.foreground);
+    public void getNextWord() {
+        Call<NextWordResponse> call = OceanTreasuresApplication.getApi().getNextWord();
 
-        RelativeLayout [] relativeLayouts = {background, foreground};
+        call.enqueue(new Callback<NextWordResponse>() {
+            @Override
+            public void onResponse(Call<NextWordResponse> call, Response<NextWordResponse> response) {
 
-        LayoutInflater Li = LayoutInflater.from(getApplicationContext());
+                if (response.code() == 404) {
 
-        for(int i = 0; i < MAX_SMALL_FISH; i++) {
-            smallFishViews.add(Li.inflate(small_fish, null));
-            relativeLayouts[generateRandomIntegerInRange(0,relativeLayouts.length)].addView(smallFishViews.get(i));
-        }
+                    AlertDialog.Builder builder = new AlertDialog.Builder(ActivityMain.this);
 
-        for(int i = 0; i < MAX_BIG_FISHES; i++) {
-            bigFishViews.add(Li.inflate(big_fish, null));
-            relativeLayouts[0].addView(bigFishViews.get(i));
-        }
-    }
+                    builder.setNeutralButton("OK", new DialogInterface.OnClickListener() {
+                        @Override
+                        public void onClick(DialogInterface dialog, int which) {
+                            MainFragment mf = new MainFragment();
+                            attachFragment(mf, "MAIN_FRAGMENT");
+                        }
+                    });
 
-    private void moveAnimation() {
-        setupAnimation(smallFishViews, smallFishBeginOffset);
-        setupAnimation(bigFishViews, bigFishBeginOffset);
-    }
+                    final AlertDialog dialog = builder.create();
+                    LayoutInflater inflater = getLayoutInflater();
+                    View dialogLayout = inflater.inflate(R.layout.alert_dialog, null);
+                    dialog.setView(dialogLayout);
+                    dialog.requestWindowFeature(Window.FEATURE_NO_TITLE);
 
-    private void setupAnimation(ArrayList<View> views, int offset) {
-        for(int i = 0; i < views.size(); i++) {
-            startAnimation(views.get(i), generateRandomIntegerInRange(MIN_DURATION, MAX_DURATION), offset);
-        }
-    }
+                    dialog.show();
 
-    private int choseRandomYPosition() {
-        int height = getScreenHeight();
```

Comment:
ScreenUtils.getScreenHeight(); - it should be obvious that a static util class is being called (Replace everywhere)